app/src/components/staking/Undelegations.vue

```diff
@@ -15,14 +15,23 @@
         <h1>
           Pending
         </h1>
-        <TableUndelegations :undelegations="undelegations" />
+        <template v-if="currentNetwork.network_type === `polkadot`">
+          <BalanceRow
+            v-for="balance in balances"
+            :key="balance.id"
+            :balance="balance"
+            :total-rewards-denom="totalRewardsDenom"
```

Comment:
this is not needed, right? let's remove together with the gql query?

Reply:
yes, it is needed to be able to display the undelegations for Polkadot. We are using this new component, `BalanceRow`

Reply:
why are rewards needed to display undelegations?

Reply:
Removed